### DISCUSSION TITLE
DocSearch doesn't work on my web site built with Docusaurus v3

### DISCUSSION THREAD

**Discussion by suzuki-shunsuke** [algolia/docsearch#2162] [General]
Hi, thank you for your great project!
I installed DocSearch on my web site but it doesn't work well.
Could you help me?

https://lintnet.github.io/

<img width="645" alt="image" src="https://github.com/algolia/docsearch/assets/13323303/921f31a3-db33-409c-a1db-0db61d126359">

> No results for "install"

I use Docusaurus v3.

https://docusaurus.io/docs/search#using-algolia-docsearch

The status code is 200.

<img width="1165" alt="image" src="https://github.com/algolia/docsearch/assets/13323303/0a068457-9a3d-4945-85b3-a6d14db8668d">

https://github.com/lintnet/lintnet.github.io/blob/bbdb9350b8b6dd34fbc6c354136a42c98b71aa91/package.json#L17-L18

https://github.com/lintnet/lintnet.github.io/blob/bbdb9350b8b6dd34fbc6c354136a42c98b71aa91/docusaurus.config.js#L108-L136

It seems I can search on Algoria Dashboard.

<img width="815" alt="image" src="https://github.com/algolia/docsearch/assets/13323303/24e4a321-e22a-496d-8b4a-ab3a6a702f87">

Indexes were created properly.

<img width="1196" alt="image" src="https://github.com/algolia/docsearch/assets/13323303/8d86e61a-60d6-4bb1-8b7e-aaec733a0fa6">

**Comment by ArthurFlag** [comment]
Hi!

I had the exact same problem, you can fix this like this:
1. Go into your dashboard **Product  > Search** (not your crawler). Click **Index**. Open the **Configuration** tab. Add these facets:
   <img width="1363" alt="Screenshot 2024-01-16 at 09 29 42" src="https://github.com/algolia/docsearch/assets/28596581/221d5af8-7057-4d51-81f0-5708c56a4a48">
2. Open your crawler config, use the [Docusaurus v3 template](https://docsearch.algolia.com/docs/templates/#docusaurus-v3-template).
3. Make sure you use the correct api key, which is the **Aloglia API key** of your crawler, not your index's.

Docsearch used to worked seamlessly out of the box with Docusaurus v2, so I'm not sure if it's some manual steps that are missing from the DocSearch team, or if it was always semi-automated and the automation needs updating, but I hope these steps help the community!

**Reply by suzuki-shunsuke** [reply]
@ArthurFlageul
Oh, thank you!
I added `Attributes for faceting` and restart crawling, then it worked!

<img width="942" alt="image" src="https://github.com/algolia/docsearch/assets/13323303/80ad4211-12f6-4f41-bb8b-93a497299718">

<img width="585" alt="image" src="https://github.com/algolia/docsearch/assets/13323303/fae3dff7-b377-4638-9093-aff7050b5da3">

**Reply by Coder6pack** [reply]
sorry ,i do like that but it can't filter 
![image](https://github.com/algolia/docsearch/assets/105229432/e4541f7b-fd4a-4745-b645-c48980629b7b)
![image](https://github.com/algolia/docsearch/assets/105229432/489ebd14-cb3a-4d61-9d7d-bcfa6932aece)
@ArthurFlag please help me

**Reply by ArthurFlag** [reply]
Send a screenshot of what you did for my first step?

**Reply by Coder6pack** [reply]
![image](https://github.com/algolia/docsearch/assets/105229432/96249340-cc6e-4cab-9b3d-7c9c3089bec9)
I add the Attributes for faceting

**Comment by ShivaAdhikari7** [comment]
It was a life saver. Thank you for sharing this.
